plugins/src/views/components/Toaster.tsx

```diff
@@ -7,15 +7,22 @@ const options: ToastOptions = {
 
 window.Toast = {
 
-  success(message) {
+  success(message, options) {
+    if (window.Pengu.silentMode)
+      return console.log(`Silent mode enabled, user won't recieve this toast`);
```

Comment:
`console.log` don't have a return value, so here you are literally returning `undefinied`, which is equivalent to not returning anything.

Also
1> Use `console.warn` would be better
2> This would cause confusion when debugging, should print the message together like `console.warn("suppressed following message since slient mode is on: %s ", "original toast message" )`